### PR TITLE
chore(*-apps): use newest argoconfig

### PR DESCRIPTION
**Pull request by eyenx**
# Description

makes our `*-apps` use the newest argoconfig version

# Issues

Fixes #291

# Checklist

* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run -a`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] If I updated a dependency tool, or app, this PR contains a short summary of the changes I'm pulling
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

Signed-off-by: Toni Tauro <toni.tauro@adfinis.com>

### CODE REVIEW

File: charts/sentry-apps/Chart.yaml [hunk 1]
```diff
@@ -14,4 +14,4 @@ maintainers:
 dependencies:
   - name: argoconfig
     repository: https://charts.adfinis.com
-    version: ~0.6.1
+    version: 0.7.4
```
Comment (hairmare) [inline charts/sentry-apps/Chart.yaml:17]: i'm kinda not sure if this one doesn't introduce any bc breakage, will have to take a closer look

Comment (tongpu) [reply charts/sentry-apps/Chart.yaml:17]: I'm not sure if sentry should still be a priority for us. I'm pushing our developers towards the cloud hosted solution right now.

Comment (eyenx) [reply charts/sentry-apps/Chart.yaml:17]: we could keep this as is, und upgrade the  rest of the chart deps. 

Btw is it a patchfix bump or minor bump?

**Comment by eyenx** [comment]
* did a minor bump of the chart version
* removed security-apps as it was already fixed by @tongpu 
* removed sentry-apps as we are unsure if it would create breaking changes

**Review by hairmare — APPROVED** [review]
LGTM since there are no more 0.6 -> 0.7 bumps involved and i'm pretty sure everything in 0.7.x is safe

**Pull request merged**